pkgs/by-name/co/cooklang-chef/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  installShellFiles,
+  gitUpdater,
+  webserver ? true,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cooklang-chef";
+  version = "0.10.2";
+
+  src = fetchFromGitHub {
+    owner = "Zheoni";
+    repo = "cooklang-chef";
+    tag = "v${version}";
+    hash = "sha256-0Cu1fkcD8l1oea35eTv+FHwqRZxIYOCy5ytkJmSTLZc=";
+  };
+
+  cargoHash = "sha256-aTIOmO6kICQgUqiA8Pjq+sfWgaObWFw4y1l1is0HTAI=";
+
+  # optionally disable the webserver
+  buildNoDefaultFeatures = !webserver;
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    # Without setting $XDG_CONFIG_HOME, `chef` will complain when trying to generate completion scripts
+    export XDG_CONFIG_HOME=$out/share
+
+    for s in bash fish zsh; do
```

```suggestion
  nativeBuildInputs = [
    installShellFiles
    writableTmpDirAsHomeHook
  ];

  postInstall = ''
    for s in bash fish zsh; do
```
so you don't pollute $out